### PR TITLE
BD-3019 Add WhatsApp SQL table references

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-3019**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 1]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+### USERS_MESSAGES_WHATSAPP_INBOUNDRECEIVE_SHARED {#USERS_MESSAGES_WHATSAPP_INBOUNDRECEIVE_SHARED}
+
+Field | Type | Description
+------|------|------------
+`id` | `string` | Globally unique ID for this event
+`time` | `int` | Unix timestamp at which the event happened
+`user_phone_number` | `string` | [PII] the user’s phone number from which the message was received
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] external user ID of the user
+`inbound_phone_number` |
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1930]:
```suggestion
`inbound_phone_number` | `string` | The inbound number that the message was sent to
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 2]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`campaign_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the campaign this event belongs to
+`campaign_api_id` | `null,`&nbsp;`string` | API ID of the campaign this event belongs to
+`message_variation_api_id` | `null,`&nbsp;`string` | API ID of the message variation this user received
+`canvas_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the Canvas this event belongs to
+`canvas_api_id` | `null,`&nbsp;`string` | API ID of the Canvas this event belongs to
+`canvas_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas variation this event belongs to
+`canvas_step_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step this event belongs to
+`canvas_step_message_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step message variation this user received
+`message_body` | `string` | Response from the user
+`quick_reply_text` | |
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1945]:
```suggestion
`quick_reply_text` | `string` | Text of button pressed by the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 3]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`campaign_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the campaign this event belongs to
+`campaign_api_id` | `null,`&nbsp;`string` | API ID of the campaign this event belongs to
+`message_variation_api_id` | `null,`&nbsp;`string` | API ID of the message variation this user received
+`canvas_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the Canvas this event belongs to
+`canvas_api_id` | `null,`&nbsp;`string` | API ID of the Canvas this event belongs to
+`canvas_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas variation this event belongs to
+`canvas_step_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step this event belongs to
+`canvas_step_message_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step message variation this user received
+`dispatch_id` | `null,`&nbsp;`string` | ID of the dispatch this message belongs to
+`message_extras` | |
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:2002]:
```suggestion
`message_extras` | `string` | The Liquid tags added to this send event 
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 4]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`campaign_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the campaign this event belongs to
+`campaign_api_id` | `null,`&nbsp;`string` | API ID of the campaign this event belongs to
+`message_variation_api_id` | `null,`&nbsp;`string` | API ID of the message variation this user received
+`canvas_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the Canvas this event belongs to
+`canvas_api_id` | `null,`&nbsp;`string` | API ID of the Canvas this event belongs to
+`canvas_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas variation this event belongs to
+`canvas_step_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step this event belongs to
+`canvas_step_message_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step message variation this user received
+`dispatch_id` | `null,`&nbsp;`string` | ID of the dispatch this message belongs to
+`message_extras` | `string` | The Liquid tags added to this send event
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:2002]:
```suggestion
`message_extras` | `null,`&nbsp;`string` | [PII] A JSON string of the tagged key-value pairs during Liquid rendering
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 5]
```diff
@@ -86,6 +86,12 @@ Table | Description
 [USERS_MESSAGES_SMS_SHORTLINKCLICK_SHARED](#USERS_MESSAGES_SMS_SHORTLINKCLICK_SHARED) | When a user clicks a Braze shortened URL included in an SMS message
 [USERS_MESSAGES_WEBHOOK_ABORT_SHARED](#USERS_MESSAGES_WEBHOOK_ABORT_SHARED) | An originally scheduled webhook message was aborted for some reason.
 [USERS_MESSAGES_WEBHOOK_SEND_SHARED](#USERS_MESSAGES_WEBHOOK_SEND_SHARED) | When we send a webhook for a user
+[USERS_MESSAGES_WHATSAPP_ABORT_SHARED](#USERS_MESSAGES_WHATSAPP_ABORT_SHARED) | An originally scheduled WhatsApp message was aborted for some reason.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:89]:
```suggestion
[USERS_MESSAGES_WHATSAPP_ABORT_SHARED](#USERS_MESSAGES_WHATSAPP_ABORT_SHARED) | An originally scheduled WhatsApp message was aborted for some reason
```
Suggestion to remove the period to match the style for the other descriptions

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 6]
```diff
@@ -705,63 +711,63 @@ Field | Type | Description
...
-`canvas_id` | `null,`&nbsp;`string` | internal-use Braze ID of the Canvas this event belongs to
+`canvas_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the Canvas this event belongs to
 `canvas_api_id` | `null,`&nbsp;`string` | API ID of the Canvas this event belongs to
 `canvas_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas variation this event belongs to
 `canvas_step_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step this event belongs to
 `canvas_step_message_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step message variation this user received
 `gender` | `null,`&nbsp;`string` | [PII] gender of the user
 `country` | `null,`&nbsp;`string` | [PII] country of the user
-`timezone` | `null,`&nbsp;`string` | timezone of the user
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:732]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 7]
```diff
@@ -705,63 +711,63 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:761]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 8]
```diff
@@ -771,35 +777,35 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:799]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 9]
```diff
@@ -809,35 +815,35 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:837]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 10]
```diff
@@ -874,218 +880,218 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:901]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 11]
```diff
@@ -874,218 +880,218 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:931]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 12]
```diff
@@ -874,218 +880,218 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:999]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 13]
```diff
@@ -874,218 +880,218 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1030]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 14]
```diff
@@ -874,218 +880,218 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1061]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 15]
```diff
@@ -874,218 +880,218 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1094]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 16]
```diff
@@ -1095,57 +1101,57 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1122]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 17]
```diff
@@ -1095,57 +1101,57 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1154]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 18]
```diff
@@ -1155,40 +1161,40 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1183]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 19]
```diff
@@ -1352,194 +1358,194 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1380]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 20]
```diff
@@ -1352,194 +1358,194 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1411]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 21]
```diff
@@ -1352,194 +1358,194 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1441]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 22]
```diff
@@ -1352,194 +1358,194 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1475]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 23]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
-`canvas_id` | `null,`&nbsp;`string` | internal-use Braze ID of the Canvas this event belongs to
+`canvas_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the Canvas this event belongs to
 `canvas_api_id` | `null,`&nbsp;`string` | API ID of the Canvas this event belongs to
 `canvas_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas variation this event belongs to
 `canvas_step_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step this event belongs to
 `canvas_step_message_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step message variation this user received
 `gender` | `null,`&nbsp;`string` | [PII] gender of the user
 `country` | `null,`&nbsp;`string` | [PII] country of the user
-`timezone` | `null,`&nbsp;`string` | time zone of the user
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1833]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 24]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+
+Field | Type | Description
+------|------|------------
+`id` | `string` | Globally unique ID for this event
+`time` | `int` | Unix timestamp at which the event happened
+`to_phone_number` | 	`null,`&nbsp;`string` | [PII] phone number of the recipient
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] External user ID of the user
+`device_id` | `null,`&nbsp;`string` | `device_id` that is tied to this user if user is anonymous
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1847]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 25]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`app_group_id` | `null,`&nbsp;`string` | ID of the workspace this user belongs to
+`app_group_api_id` | `null,`&nbsp;`string` | API ID of the workspace this user belongs to
+`subscription_group_api_id` | `string` | Subscription group API id
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1850]:
```suggestion
`subscription_group_api_id` | `string` | Subscription group API ID
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 26]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+
+Field | Type | Description
+------|------|------------
+`id` | `string` | Globally unique ID for this event
+`time` | `int` | Unix timestamp at which the event happened
+`to_phone_number` | `null,`&nbsp;`string` | [PII] phone number of the recipient
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] external user ID of the user
+`device_id` | `null,`&nbsp;`string` | `device_id` that is tied to this user if user is anonymous
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1875]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 27]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`from_phone_number` | `null,`&nbsp;`string` | Phone number from which the WhatsApp message was sent
+`app_group_id` | `null,`&nbsp;`string` | ID of the workspace this user belongs to
+`app_group_api_id` | `null,`&nbsp;`string` | API ID of the workspace this user belongs to
+`subscription_group_api_id` | `string` | Subscription group API id
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1879]:
```suggestion
`subscription_group_api_id` | `string` | Subscription group API ID
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 28]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1906]:
```suggestion
`subscription_group_api_id` | `string` | Subscription group API ID
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 29]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+
+Field | Type | Description
+------|------|------------
+`id` | `string` | Globally unique ID for this event
+`time` | `int` | Unix timestamp at which the event happened
+`to_phone_number` | `null,`&nbsp;`string` | [PII] phone number of the recipient
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] external user ID of the user
+`device_id` | `null,`&nbsp;`string` | `device_id` that is tied to this user if user is anonymous
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1902]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 30]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+Field | Type | Description
+------|------|------------
+`id` | `string` | Globally unique ID for this event
+`time` | `int` | Unix timestamp at which the event happened
+`user_phone_number` | `string` | [PII] the user’s phone number from which the message was received
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] external user ID of the user
+`inbound_phone_number` | `string` | The inbound number that the message was sent to
+`device_id` | `null,`&nbsp;`string` | `device_id` that is tied to this user if user is anonymous
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1932]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 31]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`app_group_id` | `null,`&nbsp;`string` | ID of the workspace this user belongs to
+`app_group_api_id` | `null,`&nbsp;`string` | API ID of the workspace this user belongs to
+`subscription_group_api_id` | `string` | Subscription group API id
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1935]:
```suggestion
`subscription_group_api_id` | `string` | Subscription group API ID
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 32]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`time` | `int` | Unix timestamp at which the event happened
+`to_phone_number` | `null,`&nbsp;`string` | [PII] phone number of the recipient
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] external user ID of the user
+`device_id` | `null,`&nbsp;`string` | `device_id` that is tied to this user if user is anonymous
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
+`from_phone_number` | `null,`&nbsp;`string` | Phone number from which the WhatsApp message was sent
+`app_group_id` | `null,`&nbsp;`string` | ID of the workspace this user belongs to
+`app_group_api_id` | `null,`&nbsp;`string` | API ID of the workspace this user belongs to
+`subscription_group_api_id` | `string` | Subscription group API id
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1965]:
```suggestion
`subscription_group_api_id` | `string` | Subscription group API ID
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 33]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+
+Field | Type | Description
+------|------|------------
+`id` | `string` | Globally unique ID for this event
+`time` | `int` | Unix timestamp at which the event happened
+`to_phone_number` | `null,`&nbsp;`string` | [PII] phone number of the recipient
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] external user ID of the user
+`device_id` | `null,`&nbsp;`string` | `device_id` that is tied to this user if user is anonymous
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1961]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 34]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+
+Field | Type | Description
+------|------|------------
+`id` | `string` | Globally unique ID for this event
+`time` | `int` | Unix timestamp at which the event happened
+`to_phone_number` | `null,`&nbsp;`string`	| [PII] phone number of the recipient
+`user_id` | `string` | Braze ID of the user that performed this event
+`external_user_id` | `null,`&nbsp;`string` | [PII] external user ID of the user
+`device_id` | `null,`&nbsp;`string` | `device_id` that is tied to this user if user is anonymous
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1988]:
```suggestion
`timezone` | `null,`&nbsp;`string` | Time zone of the user
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 35]
```diff
@@ -1806,28 +1812,197 @@ Field | Type | Description
...
+`from_phone_number` | `null,`&nbsp;`string` | phone number from which the WhatsApp message was sent
+`app_group_id` | `null,`&nbsp;`string` | ID of the workspace this user belongs to
+`app_group_api_id` | `null,`&nbsp;`string` | API ID of the workspace this user belongs to
+`subscription_group_api_id` | `string` | Subscription group API id
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:1992]:
```suggestion
`subscription_group_api_id` | `string` | Subscription group API ID
```

File: _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md [hunk 36]
```diff
@@ -705,63 +711,63 @@ Field | Type | Description
...
+`canvas_id` | `null,`&nbsp;`string` | Internal-use Braze ID of the Canvas this event belongs to
 `canvas_api_id` | `null,`&nbsp;`string` | API ID of the Canvas this event belongs to
 `canvas_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas variation this event belongs to
 `canvas_step_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step this event belongs to
 `canvas_step_message_variation_api_id` | `null,`&nbsp;`string` | API ID of the Canvas step message variation this user received
 `gender` | `null,`&nbsp;`string` | [PII] gender of the user
 `country` | `null,`&nbsp;`string` | [PII] country of the user
-`timezone` | `null,`&nbsp;`string` | timezone of the user
+`timezone` | `null,`&nbsp;`string` | Timezone of the user
 `language` | `null,`&nbsp;`string` | [PII] language of the user
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/segments/sql_segments/sql_segments_tables.md:762]:
```suggestion
`language` | `null,`&nbsp;`string` | [PII] Language of the user
```
Should all of the mentions of "[PII]" be "(PII)" with the first word after capitalized?

**Review by lydia-xie — APPROVED** [review]
left some suggestions and a comment on [PII] vs (PII) and capitalizing the first word after!

**Pull request merged**